dataactvalidator/app.py

```diff
@@ -39,22 +42,39 @@ def before_request():
     def handle_response_exception(error):
         """Handle exceptions explicitly raised during validation."""
         logger.error(str(error))
+        job_id = g.get('job_id', None)
```

Comment:
I like this. One risk we should watch out for, though, is documentation. We're setting a dynamic attribute in pseudo-global space; this creates an implicit protocol about the presence of that state and how to access it. Given that we're only using it in a few places, though, maybe we can just comment those lines? "job_id is set at the beginning of the validate function" or similar?